### PR TITLE
Added closed schools page for CC

### PR DESCRIPTION
**Pull request by tonyheadford**
### Context
[Trello card](https://trello.com/c/fabYSEkV/927-a-table-of-all-closed-schools-for-cc)
Add a table of closed schools to Computacenter area

### Changes proposed in this pull request
Add closed schools table showing RB and device/coms caps
Can view only those schools that can order for specific circumstances, those that can order or both.
Add link to closed schools on computacenter home page.
Added pagination to the closed school table

### Guidance to review
image
=====
image
=====
image

### CODE REVIEW

File: app/views/computacenter/home/show.html.erb [hunk 1]
```diff
@@ -21,5 +21,11 @@
     </h2>
 
     <p class="govuk-body">Download a CSV file of Google Chromebook details for schools.</p>
+
+    <h2 class="govuk-heading-m">
+      <%= govuk_link_to 'Closed schools', computacenter_closed_schools_path %>
+    </h2>
+
+    <p class="govuk-body">View details of schools that currently have Covid restrictions.</p>
```
Comment (fofr) [inline app/views/computacenter/home/show.html.erb:29]:
```suggestion
    <p class="govuk-body">View schools with closures</p>
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -82,6 +82,7 @@ en:
     computacenter:
       home: Home
       techsource: Update TechSource users
+      closed_schools: Closed schools
```
Comment (fofr) [inline config/locales/en.yml:85]: Schools with closures?

Comment (tonyheadford) [reply config/locales/en.yml:85]: Sounds good

Comment (fofr) [reply config/locales/en.yml:85]: Let's go with "School closures"

**Review by benilovj — APPROVED** [review]

**Pull request merged**